recipes/casadi/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "casadi" %}
+{% set version = "3.1.1" %}
+{% set variant = "openblas" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/casadi/casadi.git
+  git_rev: {{ version }}
+
+build:
+  number: 0
+  skip: true  # [win]
+  features:
+    - blas_{{ variant }}
+
+requirements:
+  build:
+    - cmake
+    - python
+    - swig
+    - numpy >=1.10
```

Comment:
Does this packages uses `numpy`'s ABI? If so this must be `numpy x.x` here and below.